#! /bin/sh /usr/share/dpatch/dpatch-run
## 11_manpage_bad_option.dpatch by Kevin Goodsell <kevin-opensource@omegacrash.net>
##
## DP: Remove mention of non-existent option -vo aa:help. Original patch by
## DP: Christian Bjaelevik. LP: #43301

@DPATCH@
diff -urNad mplayer-trunk~/DOCS/man/en/mplayer.1 mplayer-trunk/DOCS/man/en/mplayer.1
--- mplayer-trunk~/DOCS/man/en/mplayer.1	2010-02-04 11:33:39.000000000 -0800
+++ mplayer-trunk/DOCS/man/en/mplayer.1	2010-02-04 13:14:14.000000000 -0800
@@ -4044,15 +4044,6 @@
 .TP
 .B "aa\ \ \ \ \ "
 ASCII art video output driver that works on a text console.
-You can get a list and an explanation of available suboptions
-by executing 'mplayer \-vo aa:help'.
-.br
-.I NOTE:
-The driver does not handle \-aspect correctly.
-.br
-.I HINT:
-You probably have to specify \-monitorpixelaspect.
-Try 'mplayer \-vo aa \-monitorpixelaspect 0.5'.
 .
 .TP
 .B "caca\ \ \ "
